js/main.js

```diff
@@ -1,6 +1,139 @@
+const projects = {
+  card0: {
+    id: 0,
+    title: 'Multi-Post Stories',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    mProject: true,
+  },
+  card1: {
+    id: 1,
+    title: 'Profesional Art Printing Data',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+  card2: {
+    id: 2,
+    title: 'Profesional Art Printing Data',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+  card3: {
+    id: 3,
+    title: 'Profesional Art Printing Data',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+  card4: {
+    id: 4,
+    title: 'Profesional Art Printing Data',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+  card5: {
+    id: 5,
+    title: 'Profesional Art Printing Data',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+  card6: {
+    id: 6,
+    title: 'Profesional Art Printing Data',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+};
```

Comment:
- Kindly make sure you implement those points below to add the detail of each project in the pop-up :+1: 
   - The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.
   - Each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Ie: :+1: :smiley: 


```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```